fivetran/connectors.tf

```
@@ -54,3 +52,31 @@ resource "fivetran_connector" "github" {
     pat          = each.value.pat
   }
 }
+
+module "lambda_connector" {
+  for_each = { for lambda in var.sources_lambda : join("-", compact([lambda.service_name, lambda.project, lambda.environment])) => lambda }
+  source   = "../fivetran/connectors/lambda"
```

```suggestion
  source   = "./connectors/lambda"
```

would this short-cut work as well?